app/models/spree/vendor_translation_decorator.rb

```diff
@@ -0,0 +1,12 @@
+module Spree::VendorDecorator
+    def self.prepended(base)
+      if defined? (SpreeGlobalize)
+        base.translates :name, :about_us, :contact_us, :slug, fallbacks_for_empty_translations: true
+      end
+    end
+
+    Spree::Vendor.include SpreeGlobalize::Translatable
```

Comment:
@drkpkg you can actually put this code in https://github.com/spree-contrib/spree_multi_vendor/blob/master/app/models/spree/vendor.rb